### PR TITLE
Added handlebars support via panda-template

### LINKED ISSUE #54: Support for handlebars (via panda-template)

### PR DESCRIPTION
**Pull request by diminish7**
closes #54 

**DO NOT MERGE** Awaiting publication of `panda-9000`

TODO: Before merging: Once `panda-9000` is published with handlebars additions, change the version in `package.json`

NOTE: This is just a first step, and it doesn't currently support partials because `panda-template` doesn't. I'll add a separate issue for that.

### CODE REVIEW

File: src/survey/handlebars.coffee [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{go, map, tee, reject,
+include, Type, isType, isMatch,
```
Comment (f1337) [inline src/survey/handlebars.coffee]: `isMatch` isn't used below. I found the same in my own PR, from the copy-pasta. :smiley_cat:

Comment (diminish7) [inline src/survey/handlebars.coffee]: Yep, good call. Fixed!

File: test/fixtures/handlebars/www/page-handlebars.handlebars [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html lang="en">
+  <body>
+    <h1>Hello There</h1>
+    <p>This template had the extension .handlebars</p>
+  </body>
+</html>
```
Comment (f1337) [inline test/fixtures/handlebars/www/page-handlebars.handlebars]: We might want to include some minor handlebars expression in all 3 of these .ext templates, just to ensure the handlebars processor is actually invoked and that we're not just testing P9K's ability to rename files. :smile_cat:

Comment (diminish7) [inline test/fixtures/handlebars/www/page-handlebars.handlebars]: Yep, good call. Added.

File: README.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ Haiku9 (H9 for short) is a static site generator. H9 supports:
 
 * Jade templates
 * Markdown
+* Handlebars (via [Panda Template]()https://github.com/pandastrike/panda-template)
```
Comment (benniemosher) [inline README.md]: I think there is a small typo here. You may want the link to look like: `[Panda Template](https://github.com/pandastrike/panda-template)`

Comment (diminish7) [inline README.md]: Yep, thanks!

**Comment by f1337** [comment]
Superseded by #82. Please see the notes there re: why. 😺

**Pull request merged**